### PR TITLE
Pin fsspec s3fs gcsfs to >= 2022.1.0

### PR DESCRIPTION
**Pull request by cisaacstern**
Hopefully helps with https://github.com/pangeo-forge/pangeo-forge-recipes/issues/249

Follows @martindurant's suggestion in https://github.com/pangeo-forge/pangeo-forge-recipes/issues/249#issuecomment-982668328 (which was made prior to the release of `2022.1.0`; I figured the latest versions couldn't hurt).

Will try the slash command now to see if it solves the tutorial notebook problem discussed in the linked issue.

### CODE REVIEW

**Comment by cisaacstern** [comment]
... the slash command predictably [failed](https://github.com/pangeo-forge/pangeo-forge-recipes/runs/5060043009?check_suite_focus=true#step:4:11), as explained in https://github.com/pangeo-forge/pangeo-forge-recipes/issues/258. If the other tests pass, I will merge this, and hopefully trigger the notebook builds on merge (which according to that issue, does work).

**Comment by cisaacstern** [comment]
@rabernat, any thoughts as to why the `3.9, releases-only` tests are failing on [🤿 Bail out if no cache hit](https://github.com/pangeo-forge/pangeo-forge-recipes/runs/5060207002?check_suite_focus=true#step:6:8)?

I thought it would be good to fix #249 before releasing `0.7.0`, so we don't make a release with failing notebook tests.

**Comment by rabernat** [comment]
There are two stages to the workflow now
1. Build the environment
2. Run the test

Part 2 can't run unless the environment exists. The bail out shouldn't really happen; it indicates that the environment hasn't been built yet.

It's looking for an environment with the hash `Linux-conda-3.9-5cf9e165910c9d55b9e8c09b05c91025f01fc333d0bdb36edd564fbae34fc514-releases-only-11`
The hash should already have been built. So what's going on?

It looks like the cache is failing to store: https://github.com/pangeo-forge/pangeo-forge-recipes/runs/5060173123?check_suite_focus=true#step:17:2
```
Post job cleanup.
Unable to reserve cache with key Linux-conda-3.9-5cf9e165910c9d55b9e8c09b05c910[2](https://github.com/pangeo-forge/pangeo-forge-recipes/runs/5060173123?check_suite_focus=true#step:17:2)5f01fc333d0bdb36edd564fbae34fc514-releases-only-11, another job may be creating this cache.
```

Maybe just rerunning will fix it?

**Comment by rabernat** [comment]
I have evidently left the slash command workflow in a broken state:

https://github.com/pangeo-forge/pangeo-forge-recipes/actions/runs/1792390123

I will revert https://github.com/pangeo-forge/pangeo-forge-recipes/commit/9039dbf2b44de1d73d05d9ee94fcbe0cd8acb5ce#diff-8954df9ea60618142e08970c1b48f2319bcf78200e02ca1b17697799de1269e9

**Comment by rabernat** [comment]
Can you merge master to get f7dcdf3ef8a91daad78175a7b8c786b6b47de3ba and then try the slash command again?

**Comment by cisaacstern** [comment]
Looks like the `fsspec` upgrade got us around the issue described in #249.

We are now failing on this call to [`sst_diff.plot()`](https://github.com/pangeo-forge/pangeo-forge-recipes/runs/5069513956?check_suite_focus=true#step:13:29) with

```
ValueError: The input coordinate is not sorted in increasing order along axis 0. This can lead to unexpected results. Consider calling the `sortby` method on the input DataArray. To plot data with categorical axes, consider using the `heatmap` function from the `seaborn` statistical plotting library.
```

**Comment by rabernat** [comment]
Ok so that's actually about the data itself. I would try to run that tutorial locally on this branch and see what's going on.

**Comment by rabernat** [comment]
Charles--any update on the source of the error in the notebook? As far as I can understand, this is the main blocker for the release of 0.7, which just came up again in #282.

**Comment by cisaacstern** [comment]
> Charles--any update on the source of the error in the notebook? As far as I can understand, this is the main blocker for the release of 0.7, which just came up again in https://github.com/pangeo-forge/pangeo-forge-recipes/issues/282.

Yes, this is the release blocker.

TL;DR I haven't been able to devote time to the notebook error yet.

Longer version: I wasn't immediately sure how to dig into it, because it's some kind of xarray plotting thing, and not actually an issue with `pangeo-forge-recipes`, AFAICT. I'm sure I could figure it out given a little time, but just haven't gotten there yet.

**Comment by rabernat** [comment]
Ok so to unblock things I will merge this and go ahead and release 0.7 myself.
